capstone/src/main/webapp/group.html

```diff
@@ -3,17 +3,41 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link
       href="http://fonts.googleapis.com/css?family=Roboto"
       rel="stylesheet"
       type="text/css"
     />
     <link rel="stylesheet" href="styles/group.css"></link>
-    <script src="scripts/poll.js"></script>
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <script src="scripts/post.js"></script>
+    <script src="scripts/poll.js"></script>
     <title>Group</title>
   </head>
 
+<<<<<<< HEAD
+  <body onload="loadPosts()">
+    
+    <div class="group-content">
+
+      <div class="create-post">
+        <form id="post-form" method="POST" action="/group-post">
+          <span class="user-img align-vertical"></span>
+
+          <input type="text" id="post-input" class="align-vertical" name="post-input" value="Post a status">
+          <i class="fa fa-camera" aria-hidden="true"></i>
+           <button type="submit"><i class="fa fa-paper-plane align-vertical"></i><button>
+        </form>
+      </div>
+
+      <div class="post-feed">
+        <div id="posts-container"></div>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
Remove these, they should be added down below at the end where they were accidentally removed